### PR TITLE
Use Node to build SASS and remove Ruby

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

File: omod/pom.xml [hunk 1]
```diff
@@ -212,32 +191,14 @@
                                     <version>${uicommonsVersion}</version>
                                     <type>zip</type>
                                     <overWrite>true</overWrite>
-                                    <outputDirectory>src/main/compass/sass-external</outputDirectory>
```
Comment (mogoodrich) [inline omod/pom.xml:215]: Do we need to add "src/main/webapp/resources/styles/lib" to the gitignore as well?

Comment (mogoodrich) [reply omod/pom.xml:215]: Oh, never mind, looks like you added it in the other gitignore, I missed it.

**Review by mogoodrich — APPROVED** [review]
I'm not a node or ruby expert, but looks good to me, and I totally approve with moving from ruby to node.

**Comment by ibacher** [comment]
+1 for the watch-sass addition. Does it actually work though? My impression is that the resource is going to be served from the `target` directory rather than the `src` directory. It might make sense to have Sass directly write the compiled CSS to the appropriate place as done [here](https://github.com/openmrs/openmrs-module-referenceapplication/commit/812c5dbef79fbe7926ac02b41bc342aca17060fc). Basically just two relevant lines: [exclude the styles from the resources plugin](https://github.com/openmrs/openmrs-module-referenceapplication/commit/812c5dbef79fbe7926ac02b41bc342aca17060fc#diff-ffa8aabf43bef372dc7e0df2dc08200ed66072665a552afe214a4487c1b2ef53R87) and [tell Sass to put the output in the `target` directory](https://github.com/openmrs/openmrs-module-referenceapplication/commit/812c5dbef79fbe7926ac02b41bc342aca17060fc#diff-64c6614da250e31b39621a6e58c7b6f7343c3106523ab39e77732a832506ae5eR5).

**Comment by brandones** [comment]
Whoops, I definitely should have checked that. Thanks Ian. Working on it now.

**Comment by brandones** [comment]
@ibacher Just tested it, and it actually does work!

Peek 2021-07-26 13-38

**Pull request merged**